graph/pkg/service/v0/drives.go

```diff
@@ -272,6 +272,11 @@ func cs3StorageSpaceToDrive(baseURL *url.URL, space *storageprovider.StorageSpac
 		},
 	}
 
+	// project spaces have no single owner, therefore we don't return a value
+	if space.SpaceType == "project" {
+		drive.Owner = nil
+	}
```

Comment:
needs to shift to the Storage Space

Reply:
is `project` a fixed name? If so I would suggest to make it a constant:

i.e: `const SPACE_TYPE_PROJECT string = "project"`